linux_os/guide/system/software/integrity/software-integrity/aide/aide_scan_notification/oval/shared.xml

```diff
@@ -39,5 +50,37 @@
     <ind:pattern operation="pattern match">^.*{{{ aide_bin_path }}}[\s]*\-\-check.*\|.*/bin/mail[\s]*-s[\s]*".*"[\s]*.+@.+$</ind:pattern>
     <ind:instance datatype="int" operation="greater than or equal">1</ind:instance>
   </ind:textfilecontent54_object>
-   
+  <ind:textfilecontent54_test check="all" check_existence="all_exist"
```

Comment:
This new part isn't described in rule.yml at all. You should extend the rule description and other artifacts.

Reply:
got you will do :bow:

Reply:
OK